cmake/utils.cmake

```diff
@@ -123,18 +123,34 @@ function(combine_static_lib new_target target)
     message(FATAL_ERROR "MSVC does not yet support merging static libraries")
   endif()
 
-  set(all_libs "")
-  get_transitive_libs(${target} all_libs)
-  string(REPLACE ";" "\\\\naddlib " all_libs_string "${all_libs}")
+  # Should remove the if and use libtool on MacOS and Linux
```

Comment:
I'm just wondering if this comment is appropriate, since it's not clear to me we want to necessarily go in that direction.  Perhaps we should leave it out?